src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java

```diff
@@ -1112,4 +1112,25 @@ public void testEmptyTopic2TableMap() {
     config.put(SnowflakeSinkConnectorConfig.TOPICS, "");
     Utils.validateConfig(config);
   }
+
+  @Test
+  public void testDISABLE_REPROCESS_FILES_CLEANUP_valid_value() {
+    Map<String, String> config = getConfig();
+    config.put(SnowflakeSinkConnectorConfig.SNOWPIPE_DISABLE_REPROCESS_FILES_CLEANUP, "true");
+    Utils.validateConfig(config);
+
+    config.put(SnowflakeSinkConnectorConfig.SNOWPIPE_DISABLE_REPROCESS_FILES_CLEANUP, "False");
+    Utils.validateConfig(config);
+  }
+
+  @Test
+  public void testDISABLE_REPROCESS_FILES_CLEANUP_invalid_value() {
+    try {
+      Map<String, String> config = getConfig();
+      config.put(SnowflakeSinkConnectorConfig.SNOWPIPE_DISABLE_REPROCESS_FILES_CLEANUP, "INVALID");
+      Utils.validateConfig(config);
+    } catch (SnowflakeKafkaConnectorException exception) {
+      assert exception.getMessage().contains(SnowflakeSinkConnectorConfig.SNOWPIPE_DISABLE_REPROCESS_FILES_CLEANUP);
+    }
+  }
```

Comment:
hey nit : lets verify exception is thrown. 
The test will pass even if invalid value is passed but expected exception is not thrown. 
(`throwsException` variable that is marked true when exception is thrown, and finally we assert its true)